gulp/tasks/pldoc_scripts.js

```diff
@@ -17,3 +17,14 @@ gulp.task('pldoc_scripts', ['pldoc_scripts-lint'], function() {
         .pipe(gulp.dest(config.pldoc_dest))
         .pipe(browserSync.reload({stream:true}))
 });
+
+gulp.task('pldoc_scripts', ['pldoc_scripts-lint'], function() {
+    return gulp.src([
+        // setup sequence for files that are loaded later with RequireJS
+        './pattern-library/js/*'
+    ])
+    .pipe(uglify())
+    .pipe(gulp.dest(config.pldoc_local)) // move for just browsercync + local preview
+    .pipe(gulp.dest(config.pldoc_dest))
+    .pipe(browserSync.reload({ stream: true }))
+});
```

Comment:
@talbs I added a separate rule here to move any JS files that are later loaded indirectly with RequireJS (for example, via ui.js). We may want to use a variable for the path, but for some reason this makes my head hurt.